README.md

```diff
@@ -1,103 +1,49 @@
 
 To use pixi, install it on your machine, clone this repo, then run `pixi install` in the root of the repo.
-You may get an error big and long that mentions 'clang' or something like that. If that's the case run `export CFLAGS="-Wno-incompatible-function-pointer-types -Wno-implicit-function-declaration"` and try again. This is pre-run as a part of the setup script for the conda environment. 
+You may get an error big and long that mentions 'clang' or something like that. If that's the case run `export CFLAGS="-Wno-incompatible-function-pointer-types -Wno-implicit-function-declaration"` and try again. This is pre-run as a part of the setup script for the conda environment.
+
+To play around with the jupyter notebooks, just run `pixi run nb`, and a local instance of jupyter will be launched for you with all the necessary packages and dependencies. It's quite magic.
 
-To play around with the jupyter notebooks, just run `pixi run nb`, and a local instance of jupyter will be launched for you with all the necessary packages and dependencies. It's quite magic. 
+### DSHydro JupyterHub
+
+[The DSHydro JupyterHub](https://dshydro.ce.washington.edu/jupyter/hub) is a limited-access JupyterHub running on servers at the University of Washington.
 
-## To get running with pixi on the DSHydro environment
 1. Run `curl -fsSL https://pixi.sh/install.sh | bash` to install pixi for your user
 2. Install the data-download environment or analysis environment using `pixi install -e download-data` and `pixi install -e analysis`
 3. To look at the analysis notebook, install the analysis kernel with the following command: `./skagit-met/.pixi/envs/analysis/bin/python3 -m ipykernel install --user --name=skagit_analysis`
-4. Once installed, open the Viz.ipynb file, and select the skagit_analyis kernel. 
+4. Once installed, open the Viz.ipynb file, and select the skagit_analyis kernel.
+
+### Cloud-hosted JupyterHub:
 
-### Assumptions:
-1. I assume you have conda, and all its dependencies installed, and a base conda environment going
-2. I assume you are using bash, but if you're using zsh or another shell, this will run better if you modify setup.sh to use the environment on line 7 of setup.sh
+[Cryocloud](https://book.cryointhecloud.com/content/Getting_Started.html) is a NASA-supported JupyterHub running in AWS us-west-2. The environment setup is the same as above!
 
+### Conda environment:
+If you prefer to use conda, there is also a script to install necessary packages in a global conda environment
 1. Clone this repo
-2. In the root of the repo, run `./setup.sh` - this creates the conda environment, installs whats needed, etc.  
+2. In the root of the repo, run `./setup.sh` - this creates the conda environment, installs whats needed, etc.
 3. Enter the environment using `conda activate skagit-met`
 4. If you're done, don't forget to `conda deactivate`.
 
-# hrrr_downloader script
```

Comment:
I moved all the scripts documentation into a separate readme in the scripts/ subfolder